backend/serializers/hacknight_serializer.py

```diff
@@ -0,0 +1,17 @@
+from marshmallow import Schema, fields
+
+
+class HacknightSchema(Schema):
+    class Meta:
+        fields = ('id', 'date', 'participants')
+        dump_only = ('id',)
+
+    participants = fields.Nested(
+        'ParticipantSchema',
+        only=('id', 'github'),
+        many=True
+    )
+
+
+hacknight_schema = HacknightSchema()
```

Comment:
I personally don't see a lot of reason to have that:
```
hacknight_schema = HacknightSchema()
hacknights_schema = HacknightSchema(many=True)
```
 For a while, I was thinking that we have two separate serializers. Would prefer to use just `HacknightSchema` and `HacknightSchema(many=True)`.